src/nested_pandas/nestedframe/utils.py

```diff
@@ -0,0 +1,34 @@
+def _ensure_spacing(expr) -> str:
+    """Ensure that an eval string has spacing"""
+    single_val_operators = ["+", "-", "*", "/", "%", ">", "<", "|", "&", "~", "="]  # omit "(" and ")"
+    check_for_doubles = ["=", "/", "*", ">", "<"]
+    double_val_operators = ["==", "//", "**", ">=", "<="]
```

Comment:
I would suggest using sets for better semantics 
```suggestion
    single_val_operators = {"+", "-", "*", "/", "%", ">", "<", "|", "&", "~", "="}  # omit "(" and ")"
    check_for_doubles = {"=", "/", "*", ">", "<"}
    double_val_operators = {"==", "//", "**", ">=", "<="}
```

Reply:
Good call, implemented in latest commit